paddle/operators/shrink_rnn_memory_op.cc

```diff
@@ -46,8 +47,31 @@ class ShrinkRNNMemoryOp : public ArrayOp {
     auto *out_var = scope.FindVar(Output("Out"));
     PADDLE_ENFORCE(out_var != nullptr, "Output Out must be set");
     auto &out_tensor = *out_var->GetMutable<framework::LoDTensor>();
+
+    // should consider multiple levels
+    size_t height = dst_num_rows;
+    auto lod_level = rank_table.level();
```

Comment:
rank_table.level() is not related to ShrinkMemory.

Also, lod_level must always be zero? Since you cannot shrink the fine level of LoD without shrink the coarse level of LoD.

Reply:
Thanks, follow comments.